webservices/flow.py

```diff
@@ -0,0 +1,58 @@
+import os
+
+import networkx as nx
+
+here, _ = os.path.split(__file__)
+home = os.path.join(here, os.pardir)
+script_path = os.path.join(home, 'data', 'sql_updates')
+
+def get_graph():
+    """Build a `DiGraph` that captures dependencies between database migration
+    tasks. Each node represents a migration script, and each edge represents
+    a dependency between tasks. Tasks can be ordered using topological sort.
+    """
```

Comment:
Only if the script has dependencies or if other scripts depend on it. Adding a script like the one that generates calendar dates, which doesn't depend on any other migrations, wouldn't require a change to this file.